### PR TITLE
Code of Conduct

### PR DESCRIPTION
**Pull request by Thea-Aldrich**
This PR adds the Code of Conduct with spelling corrections and removes the enforcement clause. Enforcement and reporting procedures will be added at a later date after more input from the community.

### CODE REVIEW

**Comment by galak** [comment]
I do think we should have some contact point mentioned in here.

**Comment by carlescufi** [comment]
@Thea-Aldrich can you squash the commits? I don't think we need more than one commit here

File: CODE_OF_CONDUCT.rst [hunk 1]
```diff
@@ -13,7 +13,7 @@ religion, or sexual identity and orientation.
 
 Also refer to our
 `Zephyr project community guidelines
-<https://www.zephyrproject.org/developers/how-to-contribute/#community-guidelines>`__.
+<https://www.zephyrproject.org/developers/how-to-contribute/#community-guidelines>`
```
Comment (dbkinder) [inline CODE_OF_CONDUCT.rst]: That extra underscore was deliberate, and your patch (incorrectly) removed both of them.  Restore the original text.

**Review by dbkinder — CHANGES_REQUESTED** [review]

File: CODE_OF_CONDUCT.rst [hunk 2]
```diff
@@ -60,7 +60,7 @@ official project e-mail address, posting via an official social media
 account, or acting as an appointed representative at an online or
 offline event. Representation of a project may be further defined and
 clarified by project maintainers and `project governing bodies
-<https://www.zephyrproject.org/about/organization/>`__.
+<https://www.zephyrproject.org/about/organization/>`
```
Comment (dbkinder) [inline CODE_OF_CONDUCT.rst]: Nope.  Need the underscore to indicate this is a link (and double underscore is preferred).

**Review by dbkinder — CHANGES_REQUESTED** [review]

**Comment by Thea-Aldrich** [comment]
Replaced with #10364 https://github.com/zephyrproject-rtos/zephyr/pull/10364

**Pull request closed without merging**